SQL-Exception: Unknown column '`doktype`' in 'where clause'

The extension sounds really wonderful, I just tried to run it and got an Exception:
```
A _beUsergroups.rst.txt does already exist. Do you want to override it? (yes/no) [yes]:
A _beUsergroupTables.rst.txt does already exist. Do you want to override it? (yes/no) [yes]:
[ERROR] An exception occurred while executing a query: Unknown column '`doktype`' in 'where clause'
```

The only plausible thing I found was https://github.com/TYPO3-Documentation/project-info/blob/3f1653a2545590e23ed8c6845c6a7112492e9ec9/Classes/DataProvider/ContentCountProvider.php#L29 
However `pages.doktype` obviously exists 🤔  and a `-vvv` doesn't add a stack trace so I don't know how to help pin it down for that system as sadly I don't have the option for debugging there ¯\_(ツ)_/¯ 

Will look into it deeper as soon as times come up.

--- a/Classes/DataProvider/ContentCountProvider.php
+++ b/Classes/DataProvider/ContentCountProvider.php
@@ -82,7 +82,7 @@ class ContentCountProvider extends BaseDataProvider implements TableDataProvider
             ->from($table)
             ->where(
                 $queryBuilder->expr()->eq(
-                    $queryBuilder->quoteIdentifier($field),
+                   $field,
                     $queryBuilder->createNamedParameter($value)
                 )
             )
@@ -99,7 +99,7 @@ class ContentCountProvider extends BaseDataProvider implements TableDataProvider
             ->from($table)
             ->where(
                 $queryBuilder->expr()->like(
-                    $queryBuilder->quoteIdentifier($field),
+                    $field,
                     $queryBuilder->createNamedParameter($value)
                 )
             )

